src/Components/Alternative/Header.module.css

```diff
@@ -50,6 +50,17 @@
     font-weight: 500;
     cursor: pointer;
 }
+.mainHeaderItem a Button span
+{
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 16px;
+    font-weight: 500;
+}
```

Comment:
@HemanthPaila a good work here, but at a certain width this much font size is creating bad UX that need to be resolved
: at 797px
![image](https://user-images.githubusercontent.com/90546692/230286505-1ee3e8b3-9edb-4eda-aa29-2c0633897e61.png)

Either make a media query or make the font-size and weight this much so that it does not affect its size until hamburger icon arrives

Reply:
I apologize, @sudhanshutech, for not checking the responsiveness. I will rectify my mistake promptly.